### PR TITLE
Move console operator into a separate namespace

### PR DESCRIPTION
**Pull request by spadgett**
https://jira.coreos.com/browse/CONSOLE-1206

cc @benjaminapetersen @enj @jhadvig

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
so far so good....

**Comment by benjaminapetersen** [comment]
i think both flakes.

/retest

**Comment by spadgett** [comment]
Operator needs some additional permissions in its own namespace. Working on that now

**Comment by spadgett** [comment]
Operator and console look healthy from the logs. Looks like a flake.

/retest

**Comment by spadgett** [comment]
Tests are green

**Comment by benjaminapetersen** [comment]
Awesome.

File: manifests/04-config.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  namespace: openshift-console
+  namespace: openshift-console-operator
```
Comment (benjaminapetersen) [inline manifests/04-config.yaml]: Lots of changing the same values across the manifests.  I want something like `helm template` to be able to use `consts` across these, but hopefully wont have tons of changes in the future.

**Comment by benjaminapetersen** [comment]
🏆

**Pull request merged**